package.json

```diff
@@ -10,44 +10,30 @@
     "url": "git://github.com/bigcommerce/stencil-utils.git"
   },
   "scripts": {
-    "test": "npx grunt eslint && npx grunt karma",
-    "lint": "npx grunt lint",
-    "build": "npx grunt"
+    "build": "webpack --config webpack.conf.js",
+    "test": "jest",
+    "test-with-coverage": "jest --coverage --verbose",
+    "lint": "eslint .",
+    "lint--fix": "eslint . --fix"
   },
   "dependencies": {
     "eventemitter3": "^4.0.4",
     "whatwg-fetch": "^3.4.0"
   },
+  "browserslist": "> 0.25%, not dead",
```

Comment:
We should make this consistent with Babel target and also make sure it aligns with cornerstone config

Reply:
We can even move it out to a separate .browserslistrc file so that this config is used by all the libraries which may need it (as it's recommended here: https://babeljs.io/docs/en/babel-preset-env#browserslist-integration )